### PR TITLE
Dnsdir review

### PR DESCRIPTION
**Pull request by paulehoffman**

### CODE REVIEW

File: draft-ietf-dnsop-rfc7958bis.mkd [hunk 1]
```diff
@@ -251,7 +251,7 @@ If the relying party is using a trust anchor that has a KeyDigest element
 that does not have a `validUntil` attribute, it can change to a trust anchor
 with a KeyDigest element that does have a `validUntil` attribute,
 as long as that trust anchor's `validUntil` attribute is in the future and the
-DNSKEY elements of the KeyDigest are the same as the previous trust anchor.
+KeyTag, Algorithm, DigestType, and Digest elements of the KeyDigest are the same as the previous trust anchor.
```
Comment (GuillaumeBailey) [inline draft-ietf-dnsop-rfc7958bis.mkd:254]: Wrapping?

File: draft-ietf-dnsop-rfc7958bis.mkd [hunk 2]
```diff
@@ -414,6 +414,11 @@ retrieving the trust anchors trusts the CA that IANA uses for the
 "data.iana.org" web server, HTTPS SHOULD be used instead of HTTP in
 order to have assurance of data origin.
 
+A KeyDigest element can contain both a Digest and a PublicKey element.
+If the Digest element would not be a proper DS record for a DNSKEY record represented by the PublicKey element,
```
Comment (GuillaumeBailey) [inline draft-ietf-dnsop-rfc7958bis.mkd:418]: Wrapping?

**Review by GuillaumeBailey — APPROVED** [review]

File: draft-ietf-dnsop-rfc7958bis.mkd (hunk 2 above)
Comment (paulehoffman) [reply draft-ietf-dnsop-rfc7958bis.mkd:418]: Wrapping doesn't matter on the Markdown file: it comes out fine when running it through kramdown-rfc.

File: draft-ietf-dnsop-rfc7958bis.mkd [hunk 3]
```diff
@@ -153,7 +153,7 @@ This is in the seventh paragraph of {{xml_semantics}}.
 
 - Clarified the use of the detached CMS signature.
 
-- Added an IANA Considerations section.
+- Updated the IANA Considerations sectionto indicate requirements on IANA.
```
Comment (jschlyter) [inline draft-ietf-dnsop-rfc7958bis.mkd]: speling eror

**Review by jschlyter — CHANGES_REQUESTED** [review]

File: draft-ietf-dnsop-rfc7958bis.mkd (hunk 3 above)
Comment (paulehoffman) [reply draft-ietf-dnsop-rfc7958bis.mkd]: Fixed

**Pull request merged**